### ISSUE TITLE
How should I use ignite metrics when not using a model?

### ISSUE THREAD

**Issue by dasturge** [pytorch/ignite#2342]
## ❓ Questions/Help/Support

let's say for whatever reason, I have all my predictions already.  Maybe I've done some smoothing or postprocessing on them which had to be done after inference, or maybe I've saved them but I don't want to rerun everything and just want to test out metrics.  What's the best way to use metrics as standalones,  say as though they were a scikit learn metric?

**Comment by vfdev-5** [comment]
@dasturge Metrics can be used with Engine (using `attach` method) or without it (https://pytorch.org/ignite/metrics.html#reset-update-compute-api). So, if you have predictions and ground truth stored somewhere/somehow and let's say available as iterables, you can do something like that:
```python
import torch
from ignite.metrics import Accuracy

y_preds_iter = iter([torch.rand(4, 10) for _ in range(8)])
y_true_iter = iter([torch.randint(0, 10, size=(4, )) for _ in range(8)])

acc = Accuracy()

for y_preds, y_true in zip(y_preds_iter, y_true_iter):
    acc.update((y_preds, y_true))

print(acc.compute())
```

**Comment by vfdev-5** [comment]
@dasturge I close the question as resolved. Feel free to reopen if needed
